extensions/rich_text_components/components.py

```diff
@@ -142,7 +142,18 @@ class Link(BaseRteComponent):
 class Math(BaseRteComponent):
     """Class for Math component."""
 
-    pass
+    @classmethod
+    def validate(cls, value_dict):
+        """Validates Math component."""
+        super(Math, cls).validate(value_dict)
+        filename_pattern_regex = (
+            r'mathImg_[0-9]+_\S{10}_height_[0-9d]+_width_[0-9d]+_vertical_['
```

Comment:
Any way you can keep this constant near where the filename is constructed, or centralize the logic somehow? I worry that someone may update that logic and then forget to update this regex.

Reply:
done, added a frontend validation